package.json

```diff
@@ -4,13 +4,11 @@
   "description": "Limit the number of promises that can be in flight simultaneously",
   "main": "index.js",
   "dependencies": {
-    "chai": "^2.2.0",
-    "chai-as-promised": "^4.3.0",
     "q": "^1.2.0"
```

Comment:
Weird. I was depending on `npm outdated` but I guess that's not 100% accurate, or it doesn't do what I think it should do.. heh updating.